src/main/java/io/jenkins/plugins/sprp/PipelineSnippetGenerator.java

```diff
@@ -278,64 +262,64 @@ public String getStage(
     {
         String snippet = "stage('" + stage.getName() + "') {\n";
 
-        snippet += "\tsteps {\n";
-        snippet += "\t\t" + addTabs(getSteps(stage.getSteps()), 2);
-        snippet += "\t}\n";
+        snippet += "steps {\n";
+        snippet += getSteps(stage.getSteps());
+        snippet += "}\n";
 
         // This condition is to generate Success, Failure and Always steps after steps finished executing.
         if(stage.getFailure() != null
                 || stage.getSuccess() != null
                 || stage.getAlways() != null
                 || (stage.getName().equals("Build") &&
                         (archiveArtifacts != null || buildResultPaths != null || findbugs != null))
-                || stage.getName().equals("Tests") && (testResultPaths != null || gitConfig.getGitUrl() != null)) {
```

Comment:
Here's an example of another one of those changes that Jeff was talking about earlier.  Or just note that this is being changed in a secondary comment

Reply:
This is needed because non required post{} in the stages were generated.

Reply:
Which is why it was suggested to be in a different PR. It's not directly related to tab indentation which is what this PR is for.